content/00.front-matter.md

```diff
@@ -30,18 +30,25 @@ on {{manubot.date}}.
 + **{{author.name}}**
   {% if author.corresponding is defined and author.corresponding == true -%}^[✉](#correspondence)^{%- endif -%}
   <br>
+  {%- set has_ids = false %}
   {%- if author.orcid is defined and author.orcid is not none %}
+    {%- set has_ids = true %}
     ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
     [{{author.orcid}}](https://orcid.org/{{author.orcid}})
   {%- endif %}
   {%- if author.github is defined and author.github is not none %}
+    {%- set has_ids = true %}
     · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
     [{{author.github}}](https://github.com/{{author.github}})
   {%- endif %}
   {%- if author.twitter is defined and author.twitter is not none %}
+    {%- set has_ids = true %}
     · ![Twitter icon](images/twitter.svg){.inline_icon width=16 height=16}
     [{{author.twitter}}](https://twitter.com/{{author.twitter}})
-  {%- endif %}<br>
+  {%- endif %}
+  {%- if has_ids == true %}
```

Comment:
```suggestion
  {%- if has_ids %}
```
I haven't tested it, but I'm assuming this will work for a boolean

Reply:
I prefer `{%- if has_ids %}`, let's do that

Reply:
Done!